src/boilerplate/dispatcher.c

```diff
@@ -257,8 +266,11 @@ static void dispatcher_loop() {
         io_send_sw(SW_BAD_STATE);
     }
 
-    // call the termination callback (e.g. to return to main menu), if given
-    if (G_dispatcher_state.termination_cb != NULL) {
+    // We call the temination callback if given, but only if the UX is "dirty", that is either
```

Comment:
typo: teRmination

Reply:
Fixed in d78a9a0f8e106beae7c8bf4487234f42a0b89318.